packages/@dcl/playground-assets/etc/playground-assets.api.md

```diff
@@ -598,6 +598,7 @@ export const componentDefinitionByName: {
     "core::EngineInfo": LwwComponentGetter<LastWriteWinElementSetComponentDefinition<PBEngineInfo>>;
     "core::GltfContainer": LwwComponentGetter<LastWriteWinElementSetComponentDefinition<PBGltfContainer>>;
     "core::GltfContainerLoadingState": LwwComponentGetter<LastWriteWinElementSetComponentDefinition<PBGltfContainerLoadingState>>;
+    "core::MapPin": LwwComponentGetter<LastWriteWinElementSetComponentDefinition<PBMapPin>>;

```

Comment:
If this is a LWW, in which Entity are we going to store this information ? 
engine.RootEntity ? 
You can ONLY have ONE MapPin ?